README.md

```diff
@@ -1,33 +1,178 @@
-# @apostrophecms/require-nested-after-include 
+# @apostrophecms/stylelint-no-mixed-decls 
 
-A stylelint plugin that enforces nested rules to be defined after a `@include` rule.
+A Stylelint plugin that enforces Sass's `mixed-decls` rule — requiring declarations and nested rules to be ordered according to Sass's updated behavior.
 
-> CSS is changing the way it handles declarations mixed with nested rules,
-> and we want to make sure Sass matches its behavior.
-_https://sass-lang.com/documentation/breaking-changes/mixed-decls/_
+> ⚠️ Since Sass `1.77.0+`, CSS blocks can no longer freely mix declarations and nested rules.  
+> If you want to declare additional styles after nested rules, those declarations must be placed inside a `& { }` block.
+>
+> See: https://sass-lang.com/documentation/breaking-changes/mixed-decls/
 
-Sass will print a `mixed-decls` warning if rules following an `@include` rules are not nested in a `& { ... }` block.
-This is a breaking change in Sass since 1.77.7.
+## What this plugin does
+
+- Prevents declarations appearing after nested rules within a CSS block unless they're wrapped in a `& { }` block.
+- Handle `@include` statements that may contain nested rules.
 
 ## Installation
 
-```sh
-npm install @apostrophecms/require-nested-after-include --save-dev
+```bash
+npm install @apostrophecms/stylelint-no-mixed-decls --save-dev
 ```
 
 ## Usage
 
-If you've installed `@apostrophecms/require-nested-after-include` locally within your project, add it to the `plugins` array and activate the rule in your `stylelint` config:
+Add it to your Stylelint configuration:
 
 ```js
 {
-  "plugins": [
-    // ...
-    "@apostrophecms/require-nested-after-include"
-  ],
+  "plugins": [ "@apostrophecms/stylelint-no-mixed-decls" ],
   "rules": {
-    // ...
-    "@apostrophecms/require-nested-after-include": true
+    "@apostrophecms/stylelint-no-mixed-decls": true
+  }
+}
+```
+
+## Example: Correct Usage
+
+```scss
+.foo {
+  color: red;
+
+  &--large {
+    font-size: 24px;
+  }
+
+  & {
+    font-weight: bold;
   }
 }
 ```
+
+```scss
+@mixin foo {
+  display: block;
+  clear: both;
+}
+
+.foo {
+  @include foo;
+  color: red;
+}
+```
+
+```scss
+@mixin foo {
+  display: block;
+  clear: both;
+
+  &--large {
+    font-size: 24px;
+  }
+}
+
+.foo {
+  @include foo;
+
+  & {
+    color: red;
+  }
+}
+```
+
+```scss
+@mixin foo {
+  & {
+    display: block;
+    clear: both;
+  }
+}
+
+.foo {
+  font-weight: bold;
```

Comment:
This line shouldn't be in  $ {...} ?

Reply:
no because only declaration that follow a nested rule should also be nested.